documents/vci-flow.puml

```diff
@@ -0,0 +1,26 @@
+@startuml
+
+header VCI flow
+title
+VCI flow
+A scenario for receiving a credential in your SSI wallet
+end title
+
+autonumber
+
+participant "User" as user order 0 #ORANGE
+participant "Wallet" as wallet order 1 #RED
+participant "Demo app" as demo order 2 #BLUE
+participant "Agent" as agent order 3 #GREEN
+
+user -> demo: from the list, selects a credential type that she wants to receive
+demo -> demo: generates a form with the form data in ecosystem config file
+demo -> user: shows user the form to user.
+user -> demo: user fills in the information and clicks on send
+demo -> agent: agent receives the information from demo app and\nwith the credential template, creates a credential offer
+agent -> demo: agent sends the credential offer to the demo app
+demo -> user: demo app shows the credential offer uri in a qr code
+user -> wallet: user scans the qr code with the wallet
+wallet -> user: wallet shows user the credential the she is about to accept
```

Comment:
No proper English

Reply:
changed to `The wallet application reveals to the user the details of the credential they are about to accept.`